tests/integration/devtools/test_devtools_ui.py

```diff
@@ -110,13 +110,12 @@ def test_dvui003_callback_graph(dash_duo):
 
     # hide and redraw the callback graph so we get the new position
     dash_duo.find_element(".dash-debug-menu__button--callbacks").click()
-    dash_duo.find_element(".dash-debug-menu__button--callbacks").click()
-    sleep(2)
 
-    # fire callbacks so the callback graph redraws again
+    # fire callbacks so the profile state is regenerated
     dash_duo.find_element("#add").click()
+    dash_duo.find_element(".dash-debug-menu__button--callbacks").click()
```

Comment:
Worked fine locally but failed on CI before this commit, because the "Add" button was hidden by the callback graph. The bug still applies if the callbacks are fired while the graph is hidden, so that's what the test now does... but this brings up that it would be nice at some point to be able to shrink and/or move the callback graph so you can still interact with whichever parts of the app you want to while it's open.